spec.md

```diff
@@ -262,6 +264,8 @@ Procedure:
 SkToDpk algorithm takes a secret key SK and outputs a corresponding short formed public key.
 
 SK MUST be indistinguishable from uniformly random modulo r (Section 2.2) and infeasible to guess, e.g., generated using a trusted source of randomness.  KeyGen (Section 2.3) outputs SK meeting these requirements.  Other key generation approaches meeting these requirements MAY also be used; details of such methods are beyond the scope of this document.
+(FIXME: This paragraph is a duplicate from Section 2.2. where it is already established that IKM is random and cannot be guessed)
```

Comment:
If it is a duplicate then I think we should just remove it

Reply:
Have reviewed, @schanzen can you remove the duplicate language around SK, then we can resolve this FIX me